receivers/templates/ng_alert_notification.html

```diff
@@ -1260,7 +1260,11 @@ <h3>✅ {{ .Alerts.Resolved | len }} resolved instances</h3>
                   <tbody>
                     <tr>
                       <td align="center" class="txt" style="font-size:0px;padding:10px 25px;word-break:break-word;">
+                        {{ if .SentBy }}
+                        <div style="font-family: Inter, Helvetica, Arial; font-size: 13px; line-height: 150%; text-align: center; color: #000000;">&copy; {{ now | date "2006" }} Grafana Labs. Sent by <a href="{{ .AppUrl }}" style="color: #6E9FFF;">{{ .SentBy }}</a>.</div>
+                        {{ else }}
```

Comment:
Perhaps an easier option would be to remove the `Version` field and use `SentBy` in Grafana and Mimir. I'm fine with either approach really.